src/GoogleMapsService.php

```diff
@@ -273,7 +280,7 @@ public function newMapId(): string {
 	}
 
 	public function getResourceModules( array $params ): array {
-		return [ 'ext.maps.googlemaps3', 'ext.maps.googlemaps3ajax' ];
+		return [ 'ext.maps.googlemaps3', 'ext.maps.googlemaps3ajax', 'ext.maps.mylocation' ];
```

Comment:
If you always load the new js file anyway, you could put it in one of the existing modules.

Reply:
Sure, I'm not very familiar how script loading works in MediaWiki, but I've added it to the `ext.maps.googlemaps3` module now.